src/emc/plots/spatial.py

```diff
@@ -0,0 +1,160 @@
+__all__ = ['spatial']
+
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.colors import Normalize
+from matplotlib import rcParams, ticker, cm
+import cartopy.feature as cfeature
+import cartopy.crs as ccrs
+import matplotlib.colors as mcolors
```

Comment:
please group together with other matplotlib imports.
Might help with some consistency too e.g.
`from matplotlib import colors as mcolors`

Reply:
I realized I wasn't even using most of them so I will just delete in next commit.